provider-ui/components/ProviderComponent.js

```diff
@@ -158,108 +158,136 @@ class ProviderComponent extends React.Component {
+      session management and long-term persistence of user preferences, performance tests, service mesh adapter configurations and so on.
+      </p>
+      <List>
+      {Object.keys(availableProviders).map((key) => {
+        return (
+          <React.Fragment>
+          <ListItem
+          key={availableProviders[key]['DisplayName']}
+          className={classes.providerDesc}
+          >
+          {availableProviders[key]['Description']}
+          </ListItem>
+          </React.Fragment>
+          );                  
+      })}
+      <ListItem
+      key="SMI Tool"
+      className={classes.providerDesc}
+      >
+      Provider: SMI Tool<br />
+      <ul><li>Provider for SMI Conformance Testing</li>
+      <li>Specially create for W/G SMI Tool</li>
+      </ul>
+      </ListItem>
+
+
+      <ListItem
+      key="UT Austin"
+      className={classes.providerDesc}
+      >
+      Provider: UT Austin<br />
+      <ul><li>Curated for academic research</li>
+      <li>Provider created for students</li>
+      <li>Provides limited storage with each university domain</li>
+      </ul>
+      </ListItem>
+      </List>
+      </Typography>
+      </DialogContent>
+      <DialogActions>
+      <Button autoFocus onClick={self.handleModalClose()} color="primary">
+      OK
+      </Button>
+      </DialogActions>
+      </Dialog>
+      <div className={classes.container}>
+      <Grid item xs={12} justify="center">
+      {availableProviders !== ''
+      && (
+        <>
+        <ButtonGroup variant="contained" color="primary" ref={(ref) => self.anchorRef = ref} aria-label="split button">
+        <Button
+        size="large"
+        href={selectedProvider == '' ? '' : `/api/provider?provider=${encodeURIComponent(selectedProvider)}`}
+        >
+        {selectedProvider !== '' ? selectedProvider : 'Select Your Provider'}
+        </Button>
+        <Button
+        color="primary"
+        size="small"
+        aria-controls={open ? 'split-button-menu' : undefined}
+        aria-expanded={open ? 'true' : undefined}
+        aria-label="Select Provider"
+        aria-haspopup="menu"
+        onClick={self.handleToggle()}
+        >
+        <ArrowDropDownIcon />
+        </Button>
+        </ButtonGroup>
+        <Popper open={open} anchorEl={self.anchorRef} role={undefined} transition disablePortal>
+        {({ TransitionProps, placement }) => (
+          <Grow
+          {...TransitionProps}
+          style={{
+            transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom',
+          }}
+          >
+          <Paper>
+          <ClickAwayListener onClickAway={self.handleClose()}>
+          <MenuList id="split-button-menu">
+          {Object.keys(availableProviders).map((key) => (
+            <MenuItem
+            key={key}
+            onClick={() => self.handleMenuItemClick(key)}
+            >
+            {key}
+            </MenuItem>
+
+            ))}
+          <MenuItem disabled={true}>
```

Comment:
Please add a key here as well, even though it's disabled.

Reply:
Added MenuItem key